lib/moip2/resource/multi_order.rb

```diff
@@ -15,11 +13,9 @@ def initialize(client, response)
         end
       end
 
-      def create_multi_payment(payment)
+      def create_multi_payment(_payment)
```

Comment:
Coloca um default pra esse parametro, já que ele não é necessário. Assim quem for usar esse método já pode passar sem o parametro
`def create_multi_payment(_payment = nil)`